Makefile

```
@@ -19,9 +19,18 @@ WORKING_DIR     := $(shell pwd)
 
 OS := $(shell uname)
 
-.PHONY: development provider build_sdks build_nodejs build_go build_dotnet build_python cleanup 
+.PHONY: development regen provider build_sdks build_nodejs build_go build_dotnet build_python \
+	dev_typescript dev_go dev_python dev_dotnet lint cleanup
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)